packages/embed/src/utils/is-defined.ts

```diff
@@ -1,3 +1,7 @@
 export const isDefined = <T>(value: T | undefined | null): value is T => {
+  if (typeof value === 'string') {
```

Comment:
I would suggest against this. The purpose of this method is to check if the value is defined, which empty string is.

Reply:
what would be your suggestion? create another method?

Reply:
If it is used in multiple places then yes. Or just do `isDefined(value) && value !== ''`. Or maybe even `!!value` might work for `string | undefined`.